nemo/collections/nlp/models/language_modeling/megatron_base_model.py

```diff
@@ -293,6 +296,99 @@ def on_validation_end(self) -> None:
         if self.gc_interval > 0 and self.gc_in_validation:
             gc.collect()
 
+    def build_transformer_config(self, model_specific_config=None) -> TransformerConfig:
```

Comment:
Why do we need the model_specific_config arg?

The child class can just call this method and get the parent transformer config and then modify it on it's own from there?

Reply:
Thats a good point. DID THAT.